### PR TITLE
2532 update gems

### PR DESCRIPTION
**Pull request by arisah-ijeoma**

### CODE REVIEW

File: config/boot.rb [hunk 1]
```diff
@@ -1,6 +1,3 @@
-require 'rubygems'
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
```
Comment (emitche) [inline config/boot.rb]: What's the difference between dir and file for this line?

File: README [hunk 2]
```diff
@@ -1,261 +1,11 @@
-== Welcome to Rails
```
Comment (emitche) [inline README]: You are awesome for updating this file. Thank you!

File: config/boot.rb (hunk 1 above)
Comment (arisah-ijeoma) [reply config/boot.rb]: this came with the upgrade

Comment (emitche) [reply config/boot.rb]: Makes sense. Thanks.

**Comment by emitche** [comment]
I just need to resolve the conflicts on this and merge it into develop.

**Pull request merged**